libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -814,6 +875,29 @@ void test_MQTT_Connect_happy_path()
     status = MQTT_Connect( &mqttContext, &connectInfo, &willInfo, timeout, &sessionPresent );
     TEST_ASSERT_EQUAL_INT( MQTTSuccess, status );
     TEST_ASSERT_EQUAL_INT( MQTTConnected, mqttContext.connectStatus );
+
+    /* CONNACK receive with timeoutMs=0. Retry logic will be used. */
```

Comment:
Do we need a test case where retry logic is not used?

Reply:
The test cases above this do not use retry logic, since it's the value of the timeout that determines this